### ISSUE TITLE
phenotype_terms and subject_id tags in load config

### ISSUE THREAD

**Issue by dnil** [Clinical-Genomics/scout#2693]
Will be passed in config.yaml. phenotype_terms as an array of string, subject_id as str.

**Comment by northwestwitch** [comment]
Phenotype groups?

**Comment by dnil** [comment]
We should have a thinker about the subject IDs as well, say about automatically initially grouping cases sharing a subject. Or possibly auto-add RNA tracks for the DNA cases, if there is one of each type for the same subject.

**Comment by northwestwitch** [comment]
> We should have a thinker about the subject IDs as well, say about automatically initially grouping cases sharing a subject. Or possibly auto-add RNA tracks for the DNA cases, if there is one of each type for the same subject.

That would be very nice. But we should also consider that there might be than one of those, for instance if you rerun the RNA etc.

**Comment by dnil** [comment]
Yep.. Ideally there should be an instruction to Scout on how to handle the new RNA sample in the config from cg. Like "add" or "replace" in DNA case linking. Auto-grouping is still good for the different DNA scenarios (can always be manually removed). Also we probably need to be able to handle an RNA-only sample. Not entirely sure that would work today.
